content/docs/2.3/scalers/azure-event-hub.md

```diff
@@ -27,10 +27,19 @@ triggers:
 
```

Comment:
Can you remove the notice in the metadata please?

> notice = "As of now, the Event Hub scaler only supports reading checkpoints from Blob Storage, as well as scaling only Event Hub applications written in C#, Java, Python or created with Azure Functions."

Reply:
done